apps/openmw/mwrender/sky.cpp

```diff
@@ -1559,6 +1602,12 @@ void SkyManager::setWeather(const WeatherResult& weather)
 
             SceneUtil::DisableFreezeOnCullVisitor disableFreezeOnCullVisitor;
             mParticleEffect->accept(disableFreezeOnCullVisitor);
+
+            osgParticle::ParticleSystem *ps = (osgParticle::ParticleSystem *)
+              (mParticleEffect->asGroup()->getChild(1)->asGroup()->getChild(0)
```

Comment:
This is ugly and only works for snow. Is there a way I can find a `ParticleSystem` node in the subtree?